page.codeberg.terratactician_expandoria.game.yaml

```diff
@@ -0,0 +1,79 @@
+---
+app-id: "page.codeberg.terratactician_expandoria.game"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+command: "terratactician-expandoria"
+build-options:
+  env:
+    CARGO_HOME: "/run/build/terratactician_expandoria/cargo"
+    TTE_BUILD_ASSETS_DIR: "/app/share/terratactician_expandoria/assets"
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  # Wayland access
+  - "--socket=wayland"
+  # Sound output
+  - "--socket=pulseaudio"
+  # simply request all input devices for now
+  - "--device=all"
+  # Allow network access,
+  # required for bot API
+  - "--share=network"
+  # Allow access to pictures folder,
+  # required for screenshots
+  - "--filesystem=xdg-pictures"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "*.la"
+  - "*.a"
+modules:
+  - name: "terratactician_expandoria"
+    buildsystem: "simple"
+    build-commands:
+      # build the cargo project
+      - "cargo --offline fetch --manifest-path Cargo.toml --verbose"
+      - "cargo --offline build --release --verbose -F custom-assets-path"
+      # install binary
+      - "install -Dm755 ./target/release/terratactician-expandoria -t /app/bin/"
+      # install metainfo file
+      - "install -Dm755 ./dist/page.codeberg.terratactician_expandoria.metainfo.xml /app/share/metainfo/page.codeberg.terratactician_expandoria.game.metainfo.xml"
+      - "sed -i 's/terratactician-expandoria\\.desktop/page.codeberg.terratactician_expandoria.game.desktop/' /app/share/metainfo/page.codeberg.terratactician_expandoria.game.metainfo.xml"
+      - "sed -i 's/<id>page.codeberg.terratactician_expandoria<\\/id>/<id>page.codeberg.terratactician_expandoria.game<\\/id>/' /app/share/metainfo/page.codeberg.terratactician_expandoria.game.metainfo.xml"
+      - "sed -i 's/<icon type=\"stock\">terratactician_expandoria<\\/icon>/<icon type=\"stock\">page.codeberg.terratactician_expandoria.game<\\/icon>/' /app/share/metainfo/page.codeberg.terratactician_expandoria.game.metainfo.xml"
```

Comment:
use rename-desktop-file and rename-appdata-file to change the launchable and id. and drop stock icon from metainfo. That's not needed for GUI apps

Reply:
I've changed the `sed` icon line to remove the icon section for now. \
I'll fix it upstream as well, but I'm not sure when we are going to tag `0.2.2`